api/src/pcapi/sandboxes/scripts/creators/industrial/create_user_offerers.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.users import factories as users_factories
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_user_offerers() -> None:
+    # Add a User and UserOfferer for each Offerer without one
+
+    logger.info("create_industrial_pro_users")
+    user = None
+    count = 0
+
+    for offerer in offerers_models.Offerer.query.outerjoin(offerers_models.Offerer.UserOfferers).outerjoin(
+        offerers_models.Offerer.offererProviders
+    ):
+        # do not set a UserOfferer if there is already one or if the offerer has a provider
+        if len(offerer.UserOfferers) > 0 or len(offerer.offererProviders) > 0:
+            continue
+
+        if user == None:
+            user = users_factories.ProFactory(email="user.offerer@example.com", firstName="User", lastName="Offerer")
```

Comment:
Essayons désormais d'avoir des noms plus explicites, par exemple : 
```suggestion
            user = users_factories.ProFactory(email="user.offerer@example.com", firstName="Compte pro", lastName="Rattaché")
```